website/content/en/preview/getting-started/getting-started-with-eksctl/cloudformation.yaml

```diff
@@ -73,6 +73,13 @@ Resources:
             Action:
               - iam:PassRole
             Resource: !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/KarpenterNodeRole-${ClusterName}"
+          -  Effect: Allow
+             Action:
+              - s3:s3:Get*
+              - s3:List*
+              - s3-object-lambda:Get*
+              - s3-object-lambda:List*
+             Resource: "*"
```

Comment:
Why do we need this?

Reply:
To be able to download the AMD Driver the node needs to be able to have [AmazonS3ReadOnlyAccess](https://us-east-1.console.aws.amazon.com/iam/home#/policies/arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess)

Reply:
We definitely don't want to give these permissions to users going through this getting started guide.

Reply:
I would very much be upset if all our nodes had full read access to all S3 buckets in their AWS Account.

Reply:
Removed the read access.